### DISCUSSION TITLE
Aso's Dockyard API Feedback

### DISCUSSION THREAD

**Discussion by AsoDesu** [DockyardMC/Dockyard#78] [General]
hiya! i'm currently using and learning the dockyard api for the Ember Seeker project 
will put any pieces of feedback into here :D

**Comment by LukynkaCZE** [comment]
bro did not give any feedback 💀

**Comment by p1k0chu** [comment]
this is because dockyard is pure perfection

**Comment by AsoDesu** [comment]
feedback:
no lighting

**Reply by LukynkaCZE** [reply]
@p1k0chu I leave this feature to you thanks

**Reply by p1k0chu** [reply]
umm actually there's a cube of lighting at spawn

**Reply by LukynkaCZE** [reply]
thats just the client forcefully lighting up the chunk you spawn in in case the light wasnt received yet (so you dont get weird flashing when you first join a world)

**Reply by p1k0chu** [reply]
my fix: give everyone permanent night vision
